### DISCUSSION TITLE
OpenApi v3 import

### DISCUSSION THREAD

**Discussion by answerquest** [usebruno/bruno#547] [Ideas]
Feature request : Build an importer for importing json or yaml in OASv3 as a Collection
Specifically this: https://spec.openapis.org/oas/v3.0.3 

Saw a lack of this cited in a company as reason not to adopt Bruno yet.

So, what would this take?

**Comment by answerquest** [comment]
Ah seeing that it's already on the roadmap : https://github.com/usebruno/bruno/discussions/384
And folks used the issues section to talk about this: https://github.com/usebruno/bruno/issues/194

**Comment by nileshtrivedi** [comment]
A nice little convenience would be to be able to import a collection from a URL, instead of a local file.

**Comment by rreyn-bruno** [comment]
OpenAPI V3 is able to be imported either [from a file or from a URL.](https://docs.usebruno.com/get-started/import-export-data/import-collections)
